src/types/ScholarshipAmount.ts

```diff
@@ -108,6 +108,24 @@ namespace ScholarshipAmount {
     return { min, max, type };
   }
 
+  // Translates 'amount' to the format it should be displayed as
```

Comment:
Use JSDoc style as in the previous function (that way IDEs will display this comment when you hover over the `fromStorage` method).